source/motor.txt

```diff
@@ -17,15 +17,16 @@ Motor (Async Driver)
 Introduction
 ------------
 
-**Motor** is the recommended asynchronous Python driver for MongoDB.
-It is compatible with `Tornado <http://www.tornadoweb.org/>`__ and
-`asyncio <https://docs.python.org/3/library/asyncio.html>`__. We recommend
-that you use this driver if you need to run asynchronous operations. If
-you do not need to access MongoDB in a non-blocking manner or from
-co-routines, we recommend that you use the :doc:`PyMongo </pymongo>`
-driver instead.
-
-Follow the links below to learn more about how to use the Motor driver:
+You can use Motor, the official MongoDB driver for asynchronous Python 
+applications, to interact with MongoDB. Download it using 
+`pip <http://pypi.python.org/pypi/pip>`__
```

Comment:
Always prefer https over http links (when the TLS/SSL certificates are set up properly for the domain):

```suggestion
`pip <https://pypi.python.org/pypi/pip>`__
```

Reply:
Applied to PyMongo as well